models/vista3d/docs/labels.json

```diff
@@ -130,8 +130,5 @@
     "kidney mass": 129,
     "liver tumor": 130,
     "vertebrae L6": 131,
-    "airway": 132,
-    "FDG-avid lesion": 133,
-    "lung nodule": 134,
-    "lumbar spine": 135
+    "airway": 132
```

Comment:
@yiheng-wang-nv @KumoLiu @mingxin-zheng Is it aligned with the tutorial? I remember we decided to use some `labels.json` in the tutorial to avoid duplicated work.

Thanks.

Reply:
Hi @Nic-Ma , so far, [tutorial](https://github.com/Project-MONAI/tutorials/blob/main/3d_segmentation/vista3d/vista3d_spleen_finetune.ipynb) does not maintain a file, and it uses the label dict from VISTA repo's main branch: https://github.com/Project-MONAI/VISTA/blob/main/vista3d/data/jsons/label_dict.json

This version and the one bundle uses have some differences, thus I suggest we keep the file in this version, and change later.
I created a ticket to track this requirement: https://github.com/Project-MONAI/model-zoo/issues/664